src/controllers/update/due.diligence.controller.ts

```diff
@@ -0,0 +1,65 @@
+import { NextFunction, Request, Response } from "express";
+import { Session } from "@companieshouse/node-session-handler";
+
+import {
+  UPDATE_DUE_DILIGENCE_PAGE,
+  WHO_IS_MAKING_UPDATE_URL,
+  OVERSEAS_ENTITY_REVIEW_URL
+} from "../../config";
+
+import { ApplicationData } from "../../model";
+import { logger } from "../../utils/logger";
+import {
+  getApplicationData,
+  setApplicationData,
+  prepareData,
+  mapDataObjectToFields,
+  mapFieldsToDataObject,
+} from "../../utils/application.data";
+import { DueDiligenceKey, DueDiligenceKeys } from "../../model/due.diligence.model";
+import { OverseasEntityDueDiligenceKey } from "../../model/overseas.entity.due.diligence.model";
+import { IdentityDateKey, IdentityDateKeys } from "../../model/date.model";
+import { IdentityAddressKey, IdentityAddressKeys } from "../../model/address.model";
+import { AddressKeys, InputDateKeys } from "../../model/data.types.model";
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `${req.method} ${req.route.path}`);
+
+    const appData: ApplicationData = getApplicationData(req.session);
```

Comment:
I think we need to adopt the approach of sharing get()/post() methods with the controller for registration.

Reply:
see https://github.com/companieshouse/overseas-entities-web/pull/623/files

Reply:
We've not been consistent with the agent tickets then I guess, as UAR-101 was merged with a separate controller for update (which is why this ticket kept them separate too). Il adopt this approach but a change should be made for 101 also, at some point.

Reply:
There has been a question raised on this in #623 so please hold off combing (update/remove controllers) 
them for now.